.scripts/pr_check.sh

```diff
@@ -16,17 +16,12 @@ done
 
 script_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
-# Set credentials via env vars
-export BIGQUERY_CREDS=${BIGQUERY_CREDS:-$CREDENTIALS}
-export REDSHIFT_PASSWORD=$CREDENTIALS
-export SNOWFLAKE_PASSWORD=$CREDENTIALS
-
 echo "pr_check: Starting 10 e2e iterations"
 
 for i in {1..10}; do
   echo "pr_check: Starting e2e run $i";
 
-  bash $script_path/e2e.sh -b $SQL_RUNNER_PATH -d $DATABASE || exit 1;
```

Comment:
Does this require that we pass credentials as an argument?

I think we can solve the problem by just doing what we've done for BQ:

`export REDSHIFT_PASSWORD=${DB_PASSWORD:-CREDENTIALS}`

Which means we may use either env var or pass as argument

Reply:
The `run_config.sh`, which is called by `e2e` is already handling the credentials in this way (allow both as arg and as env vars (https://github.com/snowplow/data-models/blob/85d5fbb068783a058432b4d75f69fb2fab656b77/.scripts/run_config.sh#L48)), so i am not sure if we need to repeat here. Certainly i will look at it again.

Reply:
So we do need to set the `SNOWFLAKE_PASSWORD` and `REDSHIFT_PASSWORD` variables in our script - otherwise great_expectations will fail.

These are distinct from `DB_PASSWORD` currently, but in hindsight there's surely a better design choice there.

Reply:
Same here - let's resolve if it's been done